File: C/PRAK304-2210817220029-SitiAinurRahmawati.c
#include <stdio.h>
int main () {
    int angka;
    scanf("%d", &angka);
    if (angka>=100) {
        printf("Anda Menginput Melebihi Limit Bilangan");
    } else if (angka>10 && angka<20) {
        printf("Belasan");
    } else if (angka>=20 || angka==10) {
        printf("Puluhan");
    } else if (angka<10 && angka>0) {
        printf("Satuan");
    } else if (angka==0) {
        printf("Nol");
    }
}

File: PYTHON/PRAK304-2210817220029-SitiAinurRahmawati.py
angka=int(input())
if angka>=100:
    print("Anda Menginput Melebihi Limit Bilangan")
elif angka>10 and angka<20:
    print("Belasan")
elif angka>=20 or angka==10:
    print("Puluhan");
elif angka<10 and angka>0:
    print("Satuan")
elif angka==0:
    print("Nol")